tests/TestSuite/Mutant/ResultTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+*
+* @category   Humbug
+* @package    Humbug
+* @copyright  Copyright (c) 2015 Pádraic Brady (http://blog.astrumfutura.com)
+* @license    https://github.com/padraic/humbug/blob/master/LICENSE New BSD License
+* @author     Thibaud Fabre
+*/
+namespace Humbug\Test\TestSuite\Mutant;
+
+use Humbug\TestSuite\Mutant\Result;
+
+class ResultTest extends \PHPUnit_Framework_TestCase
+{
+    public function testGetStatusCodeReturnsCorrectCode()
+    {
+        $this->assertEquals(Result::TIMEOUT, Result::getStatusCode(true, true, true));
+        $this->assertEquals(Result::TIMEOUT, Result::getStatusCode(false, false, true));
+        $this->assertEquals(Result::TIMEOUT, Result::getStatusCode(true, false, true));
```

Comment:
As I read this test as I don't not  know Mutants real behavior I would think about Mutant that it can pass and timeout simultaneously. 
Something is not right in here.